#: __init__.py:97
msgid "NextGIS geoinformation system"
msgstr "Geoinformační systém NextGIS"

#: __init__.py:219
msgid "Linux kernel"
msgstr "Linuxové jádro"

#: __init__.py:222
msgid "OS distribution"
msgstr "Distribuce OS"

#: __init__.py:234
msgid "CPU"
msgstr "CPU"

#: __init__.py:238
msgid "RAM"
msgstr "RAM"

#: exception.py:90
msgid "Validation error"
msgstr "Chyba ověření"

#: exception.py:95
msgid "Forbidden"
msgstr ""

#: exception.py:100
msgid "Insufficient permissions"
msgstr "Nedostatečná oprávnění"

#: exception.py:105
msgid "Operational error"
msgstr "Provozní chyba "

#: exception.py:110
msgid "External service error"
msgstr ""

#: storage.py:222
msgid "Estimation is already running"
msgstr "Odhad již běží"

#: test/test_exception.py:75
#, python-format
msgid "The answer is %d"
msgstr "Odpověď je %d"

